### ISSUE TITLE
Setting 'branch' parameter to something other than 'master' needs 'outputDirectory' as well.

### ISSUE THREAD

**Issue by dayvidwhy** [KieferSivitz/vue-gh-pages#26]
When making use of the `--branch` parameter to push changes to a `gh-pages` branch, instead of adding docs in with the master branch, it attempts to make a directory at `/docs` (root of the file system) without also specifying an `--outputDirectory` parameter.

This behaviour can be seen because of;
https://github.com/KieferSivitz/vue-gh-pages/blob/master/index.js#L120

I'd be willing to help with any changes here, keen to hear your thoughts. Might just be a case of being a bit more explicit in the readme?

**Comment by KieferSivitz** [comment]
I'm pretty sure this is a bug.
I'm guessing it's [one of](https://github.com/KieferSivitz/vue-gh-pages/blob/master/index.js#L103) the [leading forward-slashes](https://github.com/KieferSivitz/vue-gh-pages/blob/master/index.js#L120) is not taking everything into account.

I will do some testing tonight to figure out which. This should be a pretty simple fix, but it is definitely a bug and not expected behavior

**Comment by KieferSivitz** [comment]
Okay, I changed the logic for the output directory to check if it was different than the default, rather than based off the branch. 

I chose to do it this way because basing the output path off the branch, rather than the output path itself was never great.

If I've missed anything or it still doesn't work let me know, it's always a possibility. 
Thanks for reporting this and the detailed info, I was able to see the problem immediately
